code/modules/mob/living/simple_animal/animals/bat.dm

```diff
@@ -1,7 +1,7 @@
 /mob/living/simple_animal/hostile/scarybat
 	name = "space bats"
 	desc = "A swarm of cute little blood sucking bats that looks pretty upset."
-	tt_desc = "Desmodus rotundus" //Common vampire bat
+	tt_desc = "N Bestia gregaria" //Common vampire bat
```

Comment:
comment is now inaccurate this is very much not the common vampire bat

Reply:
What's up with the random letters at the beginning? It's also hard to tell when the animals are actually being given their real scientific names or names or not..

Reply:
It's a planetary indicator, indicating biosphere of origin, or if they're artificial.
E = Earth
N = Nisp
A = Artificial
There were a few others but I can't seem to find the enumeration

Reply:
X is artificial, A is Aetolus. It's all in the original PRpost.